parlai/agents/bert_ranker/helpers.py

```diff
@@ -261,14 +260,14 @@ class AdamWithDecay(Optimizer):
     def __init__(
         self,
         params,
-        lr=required,
+        lr,
```

Comment:
this seems just not really necessary. we should delete the whole class.

Reply:
does torch now provide adam with decay? was HF's different?

Reply:
Yes since torch 1.2 I believe